var/spack/repos/builtin/packages/vtk-m/package.py

```diff
@@ -25,17 +25,16 @@ class VtkM(CMakePackage, CudaPackage):
     git      = "https://gitlab.kitware.com/vtk/vtk-m.git"
 
     version('master', branch='master')
+    version('1.5.5', commit="d2d1c854adc8c0518802f153b48afd17646b6252")
```

Comment:
Any particular reason for using `commit` here?  It is preferable to use the `sha256` option.

Reply:
This is the version we use as 1.5.5. Its a specific commit, there is no tarball.

Reply:
I think it may not be possible to use sha256 since there isn't a release tarball, is that right?

Reply:
My bad.  I hadn't gone out to the site to check on the release tags at the time I asked the question.

I am still bothered by the idea of externally "tagging" commits for a software project.  IMO there's potential for that to get out of sync should the developers ever decide to add tags.

BUT I also realize that this is beyond the control of people not on the software project.